### PR TITLE
Upgrade OpenResty to 1.15.8.3

### PR DESCRIPTION
**Pull request by jongiddy**
## High-level description

Upgrade OpenResty to 1.15.8.3

## Corresponding DC/OS tickets (required)

  - [D2IQ-66506](https://jira.d2iq.com/browse/D2IQ-66506) Upgrade OpenResty to 1.15.8.3

## Related tickets (optional)

  - [D2IQ-ID](https://jira.mesosphere.com/browse/D2IQ-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

File: packages/adminrouter/windows.buildinfo.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
   "sources": {
         "OpenResty": {
           "kind": "url",
-          "url": "https://openresty.org/download/openresty-1.15.8.2-win64.zip",
-          "sha1": "8379fdcb93f1bb6ace8b9f575c5fe60b831182fc"
+          "url": "https://openresty.org/download/openresty-1.15.8.3-win64.zip",
```
Comment (DominikDary) [inline packages/adminrouter/windows.buildinfo.json:5]: We should host this in our own s3 artifactory.

Comment (jongiddy) [reply packages/adminrouter/windows.buildinfo.json:5]: Sure, but I suggest that we merge this now to get merging unblocked.

**Comment by jongiddy** [comment]
@mesosphere-mergebot changelog-not-required Change log will be updated with follow-up PR

**Comment by jongiddy** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by orsenthil** [comment]
@mesosphere-mergebot merge-it

**Pull request merged**